dashboard.html

```
@@ -129,9 +129,9 @@
         <header>
             <h1>🤖 SCALPER BRAIN <span class="badge sim">SIMULATION MODE</span></h1>
             <div style="text-align: right; display: flex; flex-direction: column; gap: 5px; align-items: flex-end;">
-                <span class="status-badge active" style="background-color: #00ff00; color: #000000; font-family: monospace; font-weight: bold;">[ACTIVE] Frankfurt FX | London FX | Zurich FX | Saudi Tadawul | Frankfurt Xetra | London LSE | Euronext Paris | Johannesburg JSE | US Pre-Market | CME Futures | ICE Brent | Crypto Markets</span>
-                <span class="status-badge hold" style="background-color: #555555; color: #ffffff; font-family: monospace; font-weight: bold;">[CLOSED <= 4H] Sydney FX | Tokyo FX | Hong Kong FX | Singapore FX | Hong Kong HKEX | Shanghai SSE | Dubai DFM</span>
-                <span class="status-badge active" style="background-color: #ff9900; color: #000000; font-family: monospace; font-weight: bold;">[UPCOMING] New York FX (00:22:16) | São Paulo B3 (01:22:16) | Mexican BMV (01:22:16)</span>
+                <span class="status-badge active" style="background-color: #00ff00; color: #000000; font-family: monospace; font-weight: bold;">[ACTIVE] Frankfurt FX | London FX | Zurich FX | New York FX | Frankfurt Xetra | London LSE | Euronext Paris | Johannesburg JSE | US Pre-Market | CME Futures | ICE Brent | Crypto Markets</span>
+                <span class="status-badge hold" style="background-color: #555555; color: #ffffff; font-family: monospace; font-weight: bold;">[CLOSED <= 4H] Tokyo FX | Hong Kong FX | Singapore FX | Hong Kong HKEX | Dubai DFM | Saudi Tadawul</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)